Polynomials.Core/Polynomials.Core.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Please make it a library, and set target framework as netstandard2.1, put c# version to 8.0, and enable nullables

Reply:
Done.